mslib/msui/mscolab.py

```diff
@@ -549,17 +551,17 @@ def fetch_gravatar(self, refresh=False):
                     show_popup(self.prof_diag, "Error", "Could not create gravatar folder in config folder")
                     return
 
-            if not refresh and email_in_settings and os.path.exists(gravatar):
+            # use cached image if refresh not requested
+            if not refresh and email_in_config and os.path.exists(gravatar):
                 self.set_gravatar(gravatar)
+                return
 
-            gravatar = os.path.join(gravatar_path, f"{email_hash}.jpg")
-            gravatar_url = f"https://www.gravatar.com/avatar/{email_hash}?s=80&d=404"
+            # fetch gravatar image
+            gravatar_url = f"https://www.gravatar.com/avatar/{email_hash}.png?s=80&d=404"
```

Comment:
it would be good to make the url configurabel and also to add an option to the UI to upload an image. 
ideas can be discussed on
https://github.com/Open-MSS/MSS/issues/1287

Reply:
For this PR, I just wanted to focus on simplifying the retrieval of the gravatar images. I'll discuss ideas on the issue thread if I get any.

Reply:
sure :) that's why it is a new issue.